sound/soc/sof/intel/hda-loader-skl.c

```diff
@@ -356,7 +358,7 @@ static int cl_dsp_init_skl(struct snd_sof_dev *sdev)
 	ret = snd_sof_dsp_register_poll(sdev, HDA_DSP_BAR,
 					HDA_ADSP_FW_STATUS_SKL,
 					HDA_DSP_ROM_STS_MASK, HDA_DSP_ROM_INIT,
-					HDA_DSP_INIT_TIMEOUT,
+					chip->rom_init_timeout,
```

Comment:
@zhuyingjiang i just looked back into my PR and i've not touched the HDA_DSP_INIT_TIMEOUT but only removed the HDA_ROM_INIT_TIMEOUT. Although it looks like we should be using the rom init timeout here, I wonder why we were using HDA_DSP_INIT_TIMEOUT before. 

How did you test this? I am confused about whether this is correct or not.

Reply:
besides the register poll method has been changed recently. Can you please fix the conflicts?

Reply:
@ranj063 As I have mentioned in #704 

> Yes, SKL still need this timeout. As I test, the HDA_ROM_INIT_TIMEOUT which is 70ms is some short, So I use the already defined HDA_DSP_INIT_TIMEOUT 500ms, If this can be platform specific, 150 ms can work for SKL.

Reply:
> besides the register poll method has been changed recently. Can you please fix the conflicts?

Done.

Reply:
> @ranj063 As I have mentioned in #704
> 
> > Yes, SKL still need this timeout. As I test, the HDA_ROM_INIT_TIMEOUT which is 70ms is some short, So I use the already defined HDA_DSP_INIT_TIMEOUT 500ms, If this can be platform specific, 150 ms can work for SKL.

@zhuyingjiang sorry I missed the previous comment. Thanks